centreon/www/class/centreon-clapi/centreonLDAP.class.php

```diff
@@ -471,27 +452,46 @@ public function export($filterName = null)
             );
 
             foreach ($ldapConfigurationList as $configuration) {
-                if ($configuration['ari_name'] != 'ldap_dns_use_ssl' &&
-                    $configuration['ari_name'] != 'ldap_dns_use_tls'
+                if ($configuration['ari_name'] !== 'ldap_dns_use_ssl'
+                    && $configuration['ari_name'] !== 'ldap_dns_use_tls'
                 ) {
-                    if ($configuration['ari_name'] == 'ldap_contact_tmpl') {
+                    if ($configuration['ari_name'] === 'ldap_contact_tmpl') {
                         $contactObj = new \Centreon_Object_Contact($this->dependencyInjector);
                         $contactName = $contactObj->getParameters($configuration['ari_value'], 'contact_name');
                         $configuration['ari_value'] = $contactName['contact_name'];
                     }
                     if ($configuration['ari_name'] === 'ldap_default_cg') {
                         $contactGroupObj = new \Centreon_Object_Contact_Group($this->dependencyInjector);
                         $contactGroupName = $contactGroupObj->getParameters($configuration['ari_value'], 'cg_name');
-                        $configuration['ari_value'] = !empty($contactGroupName['cg_name'])
+                        $configuration['ari_value'] = ! empty($contactGroupName['cg_name'])
                             ? $contactGroupName['cg_name']
                             : null;
                     }
-                    echo $this->action . $this->delim . "SETPARAM" . $this->delim
+                    echo $this->action . $this->delim . 'SETPARAM' . $this->delim
                         . $ldap['ar_name'] . $this->delim
                         . $configuration['ari_name'] . $this->delim
                         . $configuration['ari_value'] . $this->delim . "\n";
                 }
             }
         }
     }
+
+    /**
+     * Checks if configuration name is unique.
+     *
+     * @param string $name
+     * @param int $arId
+     *
+     * @return bool
+     */
+    protected function isUnique($name = '', $arId = 0)
+    {
+        $stmt = $this->db->query(
```

Comment:
Please use a prepare + execute to avoid this legacy call to a modified query signature from PDO 🙏